cibuildwheel/pyodide.py

```diff
@@ -78,6 +132,8 @@ def install_xbuildenv(env: dict[str, str], pyodide_version: str) -> str:
         # PYODIDE_ROOT so copy it first.
         env = dict(env)
         env.pop("PYODIDE_ROOT", None)
+
+        # 3. Install the xbuildenv
```

Comment:
The "3" seems a bit out of place here: I don't see any 1 or 2.

Reply:
That might have been a remnant of some changes I did but removed later, fixed.